tests/gtest/integration_test_fixture.cpp

```diff
@@ -23,17 +23,11 @@ class PopulationIntegrationTestFixture : public testing::Test {
       std::stringstream ss;
         for (uint32_t i = 0; i < this->ncases_m; i++) {
```

Comment:
We are not sure if this needs to be a loop still because there aren't any looping variables in the remaining code.

Reply:
Good catch! I have removed the file since it has not been used anywhere.